examples/nft/kudos-example/token.en.shtml

```diff
@@ -0,0 +1,137 @@
+<script type="text/javascript">
+    (function() {
+        'use strict'
+        function GeneralizedTime(generalizedTime) {
+            this.rawData = generalizedTime;
+        }
+
+        GeneralizedTime.prototype.getYear = function () {
+            return parseInt(this.rawData.substring(0, 4), 10);
+        };
+
+        GeneralizedTime.prototype.getMonth = function () {
+            return parseInt(this.rawData.substring(4, 6), 10) - 1;
+        };
+
+        GeneralizedTime.prototype.getDay = function () {
+            return parseInt(this.rawData.substring(6, 8), 10)
+        };
+
+        GeneralizedTime.prototype.getHours = function () {
+            return parseInt(this.rawData.substring(8, 10), 10)
+        };
+
+        GeneralizedTime.prototype.getMinutes = function () {
+            var minutes = parseInt(this.rawData.substring(10, 12), 10)
+            if (minutes) return minutes
+            return 0
+        };
+
+        GeneralizedTime.prototype.getSeconds = function () {
+            var seconds = parseInt(this.rawData.substring(12, 14), 10)
+            if (seconds) return seconds
+            return 0
+        };
+
+        GeneralizedTime.prototype.getMilliseconds = function () {
+            var startIdx
+            if (time.indexOf('.') !== -1) {
+                startIdx = this.rawData.indexOf('.') + 1
+            } else if (time.indexOf(',') !== -1) {
+                startIdx = this.rawData.indexOf(',') + 1
+            } else {
+                return 0
+            }
+
+            var stopIdx = time.length - 1
+            var fraction = '0' + '.' + time.substring(startIdx, stopIdx)
+            var ms = parseFloat(fraction) * 1000
+            return ms
+        };
+
+        GeneralizedTime.prototype.getTimeZone = function () {
+            let time = this.rawData;
+            var length = time.length
+            var symbolIdx
+            if (time.charAt(length - 1 ) === 'Z'){
+                return 0
+            }
```

Comment:
This is still being used?

Reply:
No. It didn't integrate well with the ERC721 contract. I'll work on removing it.

Reply:
642e8c6